examples/tests/valid/bring_local.w

```diff
@@ -0,0 +1,26 @@
+bring "./store.w" as file1;
+
+// classes from other files can be used
```

Comment:
Feels like we need more tests here:

1. bring modules from subdirectories (`bring "./foo/bar/baz.w" as baz`).
2. prevent cyclic references
3. empty modules

what else?

Reply:
Added tests for bringing modules from subdirectories and empty modules. I've added an issue for handling cyclic references nicely: https://github.com/winglang/wing/issues/3509